Argo.xcodeproj/project.pbxproj

```diff
@@ -771,6 +854,53 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		D03579C81B2B7A14005D26AE /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
+				COMBINE_HIDPI_IMAGES = YES;
+				DEFINES_MODULE = YES;
+				DYLIB_COMPATIBILITY_VERSION = 1;
+				DYLIB_CURRENT_VERSION = 1;
+				DYLIB_INSTALL_NAME_BASE = "@rpath";
+				FRAMEWORK_VERSION = A;
```

Comment:
I also don't see this in the Runes diff when adding a watchOS target ... but I see this:
`TARGETED_DEVICE_FAMILY = 4;` which I think is missing from this diff